tests/spread/integration/mawk/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Integration tests for mawk
+
+execute: |
+  rootfs="$(install-slices mawk_bins)"
+
+  echo "test" | chroot "${rootfs}/" \
```

Comment:
Can you also read the original `mawk -W version` too?
We can execute that first before the functional test you have proposed.